src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -22,6 +22,9 @@
 import java.util.Arrays;
 import java.util.Locale;
 
+import javax.annotation.Nullable;
+import javax.annotation.Nonnull;
+
```

Comment:
Checker Framework does not use these annotations. It allows them (https://checkerframework.org/manual/#nullness-related-work), but I'd prefer using checker framework's Nonnull and Nullable annotations. Please see https://checkerframework.org/api/org/checkerframework/checker/nullness/qual/NonNull.html.

Also, please attentively read how the nullness checker works: https://checkerframework.org/manual/#nullness-checker

Reply:
>It treats annotations from other tools as if you had written the corresponding annotation from the Nullness Checker,

it does

> but I'd prefer using checker framework's Nonnull and Nullable annotations.

it will force us to add this tool to our compile dependecy, that will grow our jar, lets avoid this for now until we find this tool critical for us.